### DISCUSSION TITLE
Is it possible to return ipv4 only for certain hosts?

### DISCUSSION THREAD

**Discussion by ATheCoder** [DNSCrypt/dnscrypt-proxy#2651] [Q&A]
I want to have both ipv6 and ipv4. But for certain hosts such as chatgpt.com I want dnscrypt to return only the IPv4 address so my web browser is forced to connect to this website using ipv4. Is this possible with DNSCrypt-proxy?

**Comment by jedisct1** [comment]
You can run two instances. One with `block_ipv6 = false` and another one with `block_ipv6 = true`.
And use forwarding on the first one to forward specific hosts to the second instance.
